oxalis-service/src/main/java/no/difi/oxalis/service/transmission/OutboundService.java

```diff
@@ -243,6 +251,107 @@ public String sendDocument(DocumentDTO documentDTO, String userId, boolean isRes
         return transmissionIdentifier;
     }
      
+    public String sendEPEPPOLDocument(DocumentDTO documentDTO, String userId) throws IOException,
+            ClassNotFoundException, Exception {
```

Comment:
Refactor this method in to small small functions

Reply:
logic to wrap the content with header separated from the send method 
code duplication on storing evidence into the file which is removed